workflow-templates/test-package.yaml

```diff
@@ -49,9 +51,12 @@ jobs:
       - uses: dart-lang/setup-dart@v0.1
         with:
           channel: ${{ matrix.sdk }}
-      - name: Install dependencies
+      - id: install
+        name: Install dependencies
         run: dart pub get
       - name: Run VM tests
         run: dart test --platform vm
+        if: always() && steps.install.outcome == 'success'
```

Comment:
Same here: That doesn't actually change anything, if any of the steps preceding this one fail, it makes no sense to run this either. But Ok for me if this is just for symmetry.